index.js

```diff
@@ -110,26 +112,27 @@ module.exports = class Combobo {
 
   optionEvents() {
     this.cachedOpts.forEach((option) => {
-      if(!option.classList.contains('binded')){ // To check wheather the option is already in the list
-        option.classList.add('binded');
-        option.addEventListener('click', () => {
-          this
-            .goTo(this.currentOpts.indexOf(option))
-            .select();
-        });
-
-        option.addEventListener('mouseover', () => {
-          // clean up
-          const prev = this.currentOption;
-          if (prev) { Classlist(prev).remove(this.config.activeClass); }
-          Classlist(option).add(this.config.activeClass);
-          this.isHovering = true;
-        });
-
-        option.addEventListener('mouseout', () => {
-          Classlist(option).remove(this.config.activeClass);
-          this.isHovering = false;
-        });
+      // The event should not be added again for already selected options and existing options
+      if (!this.optionsWithEventHandlers.has(option.id) && !this.selected.includes(option)) {
```

Comment:
!this.selected.includes(option) : is required for not to bind the same event again for already selected options in the list.